src/components/subheader/components/descriptionBoxList/DescriptionBoxList.js

```diff
@@ -0,0 +1,58 @@
+// Library
+import PropTypes from 'prop-types'
+
+// component
+import DescriptionBox from '../descriptionBox'
+
+// readers
+import descriptionReader from '../../readers/descriptionReader'
+
+// icons
+import star from '../../../../assests/images/star.svg'
+import time from '../../../../assests/images/time.svg'
+import rupee from '../../../../assests/images/rupee.svg'
+
+
+function DescriptionBoxList(props){
+
+    const { restaurantInfo } = props;
+
+    const rating = descriptionReader.rating(restaurantInfo)
+    const ratingCount = descriptionReader.ratingCount(restaurantInfo)
+    const deliveryTime =  descriptionReader.deliveryTime(restaurantInfo)
+    const cost = descriptionReader.cost(restaurantInfo)
+    const peopleCount = descriptionReader.peopleCount(restaurantInfo)
+
+    return(
+        <div className="flex">
+            <DescriptionBox icon={star} heading={rating} subtitle={`${ratingCount}+ Ratings`}/>
+            <DescriptionBox icon={time} heading={deliveryTime} subtitle="For delivery"/>
+            <DescriptionBox icon={rupee} heading={cost} subtitle={`Cost for ${peopleCount}`}/>
+        </div>
+    )
+}
+
+DescriptionBoxList.propTypes = {
+    restaurantInfo : PropTypes.arrayOf(PropTypes.object)
```

Comment:
propTypes and defaultProps doesn't match

Reply:
You can check the console for these types of error